go/src/koding/kites/metrics/cron.go

```diff
@@ -0,0 +1,65 @@
+package metrics
+
+import (
+	"koding/kites/kloud/metrics"
+	"time"
+
+	"github.com/koding/logging"
+)
+
+type kloudKite interface {
+	Call(method string, req, resp interface{}) error
+}
+
+// StartCron starts klient metrics publisher.
+func StartCron(kloud kloudKite, log logging.Logger) {
+	for range time.NewTicker(time.Second * 3).C {
+		process(kloud, log)
+	}
+}
+
+// StartCronWithMetrics starts klient metrics publisher with given db.
+func StartCronWithMetrics(kloud kloudKite, log logging.Logger, m *Metrics) {
+	for range time.NewTicker(time.Second * 3).C {
+		processWithMetrics(kloud, log, m)
+	}
+}
+
+func process(kloud kloudKite, log logging.Logger) {
+	m, err := New("kd")
+	if err != nil {
+		log.Error(err.Error())
```

Comment:
log.Error("%s", err)

If err.Error() contains `%` characters, the log line will be misformatted.